Pidaq/UI/Stability/stability_dash/layout/control.py

```diff
@@ -0,0 +1,151 @@
+import dash
+import dash_bootstrap_components as dbc
+import dash_daq as daq
+import dash_core_components as dcc
+import dash_html_components as html
+
+# Main Control GUI Bootstrap Layout
+control_layout = html.Div([
+
+dbc.Row([
```

Comment:
Add Min/Max constants for each input at the top of this file. 

OR we can add another file like suspension_config.py that includes all these constants and then import it. I kind of like that a little better!